user_sync/rules.py

```diff
@@ -140,7 +153,26 @@ def run(self, directory_groups, directory_connector, dashboard_connectors):
         self.clean_dashboard_users(dashboard_connectors)    
         dashboard_connectors.execute_actions()
         dashboard_stats.log_end(logger)
+        self.log_action_summary()
             
+    def log_action_summary(self):
+        '''
+        log number of directory and Adobe users, number of created / updated / deleted users, and number of users with updated groups
+        or removed from mapped group because they were not in directory
+        :return: None
+        '''
+        logger = self.logger
+        # find the total number of directory users
+        self.action_summary['number_of_directory_users'] = len(self.filtered_directory_user_by_user_key);
```

Comment:
You need to report both the total number of directory users and the number of directory users in the "selected" input set (i.e., filtered set).